pkg/install/install.go

```diff
@@ -64,7 +66,9 @@ func Kourier() error {
 	}
 	fmt.Println("    Ingress patched...")
 
-	config := `apiVersion: v1
+	// special config for different local environments
+	if k8sType == "kind" {
```

Comment:
Instead of passing the types in, leading to a leaky abstraction, should we move the type specific thing into the `SetUp` function of the respective type?